packages/kbn-search-api-panels/components/code_box.tsx

```diff
@@ -89,7 +89,7 @@ export const CodeBox: React.FC<CodeBoxProps> = ({
 
   return (
     <EuiThemeProvider colorMode="dark">
-      <EuiPanel paddingSize="xs" className="serverlessSearchCodeBlockControlsPanel">
```

Comment:
🤔 I think the css for this got renamed and we never updated the className here. the `serverlessSearchCodeBoxPanel` is defined in `code_box.scss` but not used anywhere anymore.

Reply:
if we remove this we can remove the scss file too. I'd have to test and see how the class changes the rendering to be sure.